4.7:MicoAir743-AIO (#6511)
ArduPilot/ardupilot_wiki: common/source/docs/common-MicoAir743-AIO.rst

[copywiki destination="plane,copter,rover,blimp"]

==============
MicoAir743-AIO
==============

The MicoAir743-AIO is a flight controller designed and produced by `MicoAir Tech <http://micoair.com/>`_.

Features
========

* STM32H743 microcontroller
* BMI088/BMI270 dual IMUs
* DPS310 barometer
* 12V 2A BEC; 5V 2A BEC
* MicroSD Card Slot
* 7 UARTs
* Integrated 4-in-1 AM32 35A ESC, 5 additional PWM outputs, one defaulted as serial LED
* 1 I2C
* 1 SWD

.. image:: ../../../images/MicoAir743-AIO_Physical_Size.jpg
   :target: ../_images/MicoAir743-AIO_Physical_Size.jpg
   :width: 50%

Connectors
==========

.. image:: ../../../images/MicoAir743-AIO_Ports_Connection.jpg
   :target: ../_images/MicoAir743-AIO_Ports_Connection.jpg

UART Mapping
============

* SERIAL0 -> USB
* SERIAL1 -> UART1 (MAVLink2, DMA-enabled)
* SERIAL2 -> UART2 (DisplayPort, DMA-enabled)
* SERIAL3 -> UART3 (GPS, DMA-enabled)
* SERIAL4 -> UART4 (MAVLink2, DMA-enabled)
* SERIAL5 -> UART6 (RCIN, DMA-enabled)
* SERIAL6 -> UART7 (RX only, ESC Telemetry, DMA-enabled)
* SERIAL7 -> UART8 (User, DMA-enabled)

Serial protocols shown are defaults, but can be adjusted to personal preferences.

RC Input
========

> The default RC input is configured on the UART6 RX6 input and can be used for all ArduPilot supported unidirectional receiver protocols. The SBus input on the HD VTX connector is tied to RX6 so do not connect anything simultaneously with anything else on RX6.

Reviewer: ```suggestion
The default RC input is configured on the UART6 RX6 input and can be used for all ArduPilot supported unidirectional receiver protocols. The SBus input on the HD VTX connector is tied to RX6 so do not connect anything on RX6 if using HD VTX.
```
is maybe a little clearer

Author: I reworded, thanks
